requirements.txt

```diff
@@ -10,4 +11,8 @@ thop
 ninja
 tabulate
 tensorboard
-onnxruntime
+
+# verified versions
+onnx==1.8.1
+onnx-simplifier==0.3.5
+onnxruntime-gpu==1.8.0
```

Comment:
@Joker316701882 PTAL

Reply:
onnxruntime-gpu seems based on a certain CUDA version, hence I think it is not necessary. Besides, onnx-simplier is somehow relevant to onnxruntime. So the order of onnx-simplifier and onnxruntime should be reversed. So I suggest:

onnx==1.8.1
onnxruntime==1.8.0
onnx-simplifier==0.3.5

Reply:
Noted

Reply:
Seems no updates?

Reply:
Updated

Reply:
plz update as @Joker316701882 suggested, thx

Reply:
Updated